tests/gold_tests/timeout/create_request.sh

```diff
@@ -16,5 +16,42 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-printf GET / HTTP/1.1
-sleep 11
+# Different vresions of nc have different wait options,
+# so falling back to shell
+
+# Send part of the request and then stop
+request() {
+	printf "GET / HTTP/1.1"
+}
+
+nc_name="nc"
+ncat --help
+if [ $? -le 1 ]
```

Comment:
I'm surprised by `-le 1` rather than `-eq 0`. Do some `ncat` commands return `1` with `--help`? I assume checking for `1` is intentional, but just verifying.

Related to this, we might want to consider using the built-in `command -v ncat` instead of `ncat --help`.